src/group_theory/solvable.lean

```diff
@@ -175,4 +227,39 @@ begin
   ... = ⊥ : rfl,
 end
 
+lemma is_solvable_of_top_eq_bot (h : (⊤ : subgroup G) = ⊥) : is_solvable G :=
```

Comment:
This means that `G` is a `subsingleton`, right? So you could turn this into an instance: `[subsingleton G] : is_solvable G`.

Reply:
I think it's slightly more convenient for us to have an instance whose assumption is top equals bottom so I've just added another instance which assumes `subsingleton G` instead of replacing this one. Is that okay? If not, I'm happy to replace this instance instead.